src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -0,0 +1,1463 @@
+        // more flexible, we'll make it so.
+        return "A1";
+    }
+
+    private static PartyIdentification32 constructCreditor(
+            final PaymentGroup templatePaymentGroup,
+            final ExtractTypeContext extractTypeContext
+    ) {
+        final String creditorName = templatePaymentGroup.getPayeeName();
+        final PartyIdentification32 creditor = constructPartyIdentification(creditorName);
+
+        final PostalAddress6 creditorPostalAddress = constructPostalAddress(templatePaymentGroup);
+        creditor.setPstlAdr(creditorPostalAddress);
+
+        final Party6Choice creditorId = constructCreditorId(templatePaymentGroup, extractTypeContext);
+        creditor.setId(creditorId);
+
+        return creditor;
+    }
+
+    private static Party6Choice constructCreditorId(
+            final PaymentGroup templatePaymentGroup,
+            final ExtractTypeContext extractTypeContext
+    ) {
+        final String vendorNumber = StringUtils.truncate(templatePaymentGroup.getPayeeId(), VENDOR_NUM_MAX_LENGTH);
+        final GenericOrganisationIdentification1 otherId = new GenericOrganisationIdentification1();
+        otherId.setId(vendorNumber);
+        if (extractTypeContext.isExtractionType(ExtractionType.ACH)) {
+            final String schemaNameCode = "CHID";
+            final OrganisationIdentificationSchemeName1Choice schemaName =
+                    new OrganisationIdentificationSchemeName1Choice();
+            schemaName.setCd(schemaNameCode);
+
+            otherId.setSchmeNm(schemaName);
+        }
+
+        final OrganisationIdentification4 creditorOrganizationId = new OrganisationIdentification4();
+        creditorOrganizationId.addOthr(otherId);
+
+        final Party6Choice creditorId = new Party6Choice();
+        creditorId.setOrgId(creditorOrganizationId);
+        return creditorId;
+    }
+
+    private RemittanceInformation5 constructRemittanceInformation(
+            final PaymentGroup templatePaymentGroup,
+            final Collection<PaymentDetail> paymentDetails,
+            final ExtractTypeContext extractTypeContext
+    ) {
+        final RemittanceInformation5 remittanceInformation = new RemittanceInformation5();
+
+        /*
+         * CU Customization: Removed the setup of unstructured remittance information.
+         */
+        //final String ustrd = constructUstrd(templatePaymentGroup);
+        //remittanceInformation.addUstrd(ustrd);
+
+        paymentDetails.forEach(paymentDetail -> {
+            final StructuredRemittanceInformation7 structuredRemittanceInformation =
+                    constructStructuredRemittanceInformation(paymentDetail, extractTypeContext);
+            remittanceInformation.addStrd(structuredRemittanceInformation);
+        });
+
+        return remittanceInformation;
+    }
+
+    /*
+     * CU Customization: Marked this method as "unused" since the customized code no longer calls it.
+     */
+    @SuppressWarnings("unused")
```

Comment:
I am not sure the best approach but I am wondering if it would be better to comment out or remove the function entirely?  I am wondering what is the best way to help someone doing a kualico patch update to notice we don't use this function